homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/set/UniqueWordVocabulary.java

```diff
@@ -0,0 +1,54 @@
+package com.tutrit.java.quickstart.homework.set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.LinkedHashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class UniqueWordVocabulary {
+
+    Logger log = LoggerFactory.getLogger("Set ");
+
+    Set<String> linkedHashSetSet = new LinkedHashSet<>();
+
+    public void addWord(String word) {
+        if (!word.equals("")) {
+            linkedHashSetSet.add(word);
+        }
+    }
+
+    public int getWordsCount() {
+        return linkedHashSetSet.size();
+    }
+
+    public void printVocabulary() {
+        for (String set : linkedHashSetSet
+        ) {
```

Comment:
you'd better not split braces, to prevent such mistypes please use sonar and ctrl + shift +alt + L

Reply:
I do not know why, but the formatting of the code by the Idea did not fix it. Fixed.